vars/buildNodeJs.groovy

```diff
@@ -80,8 +80,10 @@ void call(Map config=[:]) {
           utils.replaceInFile(containerSrcFolder, localSrcFolder, lcovFile)
         }
 
-        stage('Publish pact broker') {
-          pact.publishContractsToPactBroker(repoName, version.getPackageJsonVersion(), utils.getCommitSha())
+        if (pr == '') {
+          stage('Publish pact broker') {
+            pact.publishContractsToPactBroker(repoName, version.getPackageJsonVersion(), utils.getCommitSha(), defaultBranch)
```

Comment:
The `defaultBranch` parameter isn't used so can be removed.

Reply:
Good spot, forgot to remove from previous changes.